tests/monotouch-test/CoreFoundation/ProxyTest.cs

```diff
@@ -76,6 +76,9 @@ public void Setup ()
 							var context = listener.GetContext ();
 							var request = context.Request;
 							var pacPath = Path.Combine (NSBundle.MainBundle.ResourcePath, request.RawUrl.Substring (1));
+							pacPath = Path.GetFullPath (pacPath);
+							if (!pacPath.StartsWith (Path.GetFullPath (NSBundle.MainBundle.ResourcePath) + Path.DirectorySeparatorChar))
+								throw new InvalidOperationException ($"Invalid access: {request.RawUrl}");
```

Comment:
Does it make sense to put "is not within the resource path"?

Reply:
This is for a test, no human is likely to ever see this unless something goes quite wrong, so IMHO it's not worth the extra cycles to change it now.